maraboupy/MarabouCore.cpp

```diff
@@ -317,6 +341,7 @@ InputQuery loadQuery(std::string filename){
 // Describes which classes and functions are exposed to API
 PYBIND11_MODULE(MarabouCore, m) {
     m.doc() = "Maraboupy bindings to the C++ Marabou via pybind11";
+    m.def("loadProperty", &loadProperty, "Create input query from property file");
```

Comment:
It's not really creating an IPQ, right? Just adding the property on top of an existing IPQ?